dev_guide/volumes.adoc

```diff
@@ -130,7 +130,7 @@ character.
 |`--source`
 |Details of volume source as a JSON string. Recommended if the desired volume
 source is not supported by `--type`. See
-link:https://github.com/openshift/origin/blob/master/Godeps/_workspace/src/github.com/GoogleCloudPlatform/kubernetes/pkg/api/types.go#L179[available
+link:https://docs.openshift.org/latest/rest_api/kubernetes_v1.html#v1-volume[available
```

Comment:
@ashcrow This would probably be better as a relative link, since the source is used to built out both the Enterprise and Origin docs:

```
link:../rest_api/kubernetes_v1.html#v1-volume[available volume sources]
```